src/extensions/common/hither/useHitherJob.ts

```diff
@@ -20,6 +19,10 @@ const useHitherJob = <T>(functionName: string, functionArgs: {[key: string]: any
         return true
     }
 
+    if (!functionName) {
```

Comment:
Hooks cannot be conditionally run. But sometimes the input is not yet ready. This condition here gives us a way to tell the hook that we are not ready with the inputs. If the functionName is an empty string, we just return an undefined result.